packages/nc-gui/components/workspace/Settings.vue

```diff
@@ -139,22 +141,65 @@ const onCancel = () => {
         </div>
       </a-form>
     </div>
-    <div class="item flex flex-col">
+    <div class="item flex flex-col border-1 border-red-500">
       <div class="font-medium text-base">Delete Workspace</div>
       <div class="text-gray-500 mt-2">Delete this workspace and all it’s contents.</div>
-      <div class="flex flex-row mt-8 gap-x-2">
-        <a-checkbox v-model:checked="isConfirmed" />
-        <div class="flex">I understand that this action is irreversible</div>
-      </div>
-
+      <!-- <div class="flex p-4 border-1 border-[#E7E7E9] rounded-lg mt-6">
+        <div class="flex">
+          <component :is="iconMap.warning" class="w-6 h-6 text-[#FA8231]" />
+          <div class="ml-4 flex flex-col">
+            <h4 class="font-medium text-base">This action is irreversible</h4>
+            <div class="text-gray-500 mt-1">
+              You have 31 days to undo this action by following the steps provided in recover workspace mail sent to your email.
+            </div>
+          </div>
+        </div>
+      </div> -->
       <div class="flex flex-row w-full justify-end mt-8">
-        <NcButton type="danger" :disabled="!isConfirmed" :loading="isDeleting" @click="onDelete">
-          <template #loading> Deleting Workspace </template>
+        <NcButton
+          type="danger"
+          @click="
+            () => {
+              workspaceToDelete = activeWorkspace?.title
+              isModalVisible = true
+            }
```

Comment:
This is not a good practice, add a function to script part instead.
Also I would prefer resetting form input on closing and reopening.